test/HLSL/passes/instcombine/disable-bswap-match.ll

```diff
@@ -0,0 +1,89 @@
+; RUN: opt -instcombine -S %s | FileCheck %s
```

Comment:
Maybe put into test\HLSLFileCheck\passes.
Things under test/HLSL will not be tested automatically.

Reply:
I fixed that in https://github.com/microsoft/DirectXShaderCompiler/pull/5887. These tests now run automatically as expected.

Reply:
This test isn’t in that directory. This is `test/HLSL`, not `clang/test/HLSL`

Reply:
I see. :( Sorry about it.
If that's the case, then maybe test/DXIL is better? Since is not about HLSL but DXIL limitation?

Reply:
The right place for this test (and the other tests in the `test/HLSL` folder) is under `Transforms`, but we currently have that directory disabled. We should fix the `Transforms` suite, then move these under. I don't think moving these to a `DXIL` directory is necessarily helpful because they aren't tests for `lib/DXIL` they're tests for `lib/Transform`.